f8a_worker/workers/report_generation.py

```diff
@@ -0,0 +1,122 @@
+"""Output: TBD."""
+
+from f8a_worker.base import BaseTask
+from f8a_worker.graphutils import GREMLIN_SERVER_URL_REST
+from f8a_worker.utils import MavenCoordinates
+from f8a_worker.utils import get_session_retry
+import json
+import operator
+from time import gmtime, strftime
+import semantic_version as sv
+
+
+class ReportGenerationTask(BaseTask):
+    """Generates report containing descriptive data for dependencies."""
+
+    def parse_version_data(self, data):
+        """
+        Parse version data extracted from graph for a given dependency.
+
+        :param data: version data extracted from graph
+        :return: parsed version data
+        """
+        self.log.info("Version data received is: {}".format(data))
+        version_data = dict()
+        version = data.get('version')[0]
+        cve_ids = data.get('cve_ids', [])
+        licenses = data.get('licenses', [])
+        cve_list = list()
+        for cve_id in cve_ids:
+            cve_name, cvss_score = cve_id.split(":")
+            cve_list.append((cve_name, float(cvss_score)))
+        cve_list.sort(key=operator.itemgetter(1), reverse=True)
+        version_data.update({"version": version, "cve_count": len(cve_ids), "licenses": licenses})
```

Comment:
You don't need `version_data` before this point, so you can define it here:
```python
version_data = {"version": version, "cve_count": len(cve_ids), "licenses": licenses}
```

Reply:
👍